core/static/js/dit.class.modal.js

```diff
@@ -28,7 +28,8 @@
       $activators: $(), // (optional) Element(s) to control the Modal
       closeOnBuild: true, // Whether intial Modal view is open or closed
       overlay: true,  // Whether it has an overlay or not
-      closeButtonId: '' // Option to add custom close button id
+      closeButtonId: '', // Option to add custom close button id
+      onClose: function() {}
```

Comment:
Best to add an explanatory comment like for the other config options e.g. "Option to add custom function for when modal is closed"